monai/data/image_writer.py

```diff
@@ -716,3 +782,15 @@ def create_backend_obj(
             data = np.moveaxis(data, 0, 1)
 
         return PILImage.fromarray(data, mode=kwargs.pop("image_mode", None))
+
+
+def init():
+    """
+    Initialize the image writer modules according to the filename extension.
+    """
+    for ext in (".png", ".jpg", ".jpeg", ".bmp", ".tiff", ".tif"):
```

Comment:
Here the `ext` contains "." at the beginning, while in readers we don't contain it:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/image_reader.py#L393
I feel it's better to be consistent?

Thanks.